concourse/pipelines/create-bosh-cloudfoundry.yml

```diff
@@ -1258,6 +1232,44 @@ jobs:
 
                   cf push
 
+      - task: deploy-healthcheck
```

Comment:
This one is only in `create-bosh-cloudfoundry.yml` so there is no code duplication. I'm not sure about moving it to separate task as we prefer to have tasks inline if possible ( and reasonable ).